packages/owlbot-bootstrapper/common-container/test/github-authenticator.test.ts

```diff
@@ -74,37 +75,61 @@ describe('behavior of Github Authenticator Class', async () => {
   });
 
   it('should get a short lived access token', async () => {
-    const githubAuthenticator = new GithubAuthenticator(
-      'projectId',
-      '2345567',
-      secretManagerClientStub
+    const githubAuthenticatorStub = sinon.createStubInstance(
+      GithubAuthenticator
+    ) as SinonStubbedInstance<GithubAuthenticator> & GithubAuthenticator;
+
+    githubAuthenticatorStub.getGitHubShortLivedAccessToken.callsFake(
+      async () => {
+        const resp = await request({
+          url: 'https://api.github.com/app/installations/2345567/access_tokens',
+          method: 'POST',
+          headers: {
+            Authorization: 'Bearer 1234',
+            Accept: 'application/vnd.github.v3+json',
+          },
+        });
+        return (resp as any).data.token as string;
```

Comment:
I don't think this refactor makes sense to do -- it's testing its own fake. This replaces the `getGitHubShortLivedAccessToken` with a fake that makes and API call that's faked.

Reply:
You're right. I couldn't think of a better solution but the break helped me come up with something better. Thanks!